src/crdt_bcounter.erl

```diff
@@ -154,6 +155,28 @@ to_binary(C) -> term_to_binary(C).
 -spec from_binary(binary()) -> bcounter().
 from_binary(<<B/binary>>) -> binary_to_term(B).
 
+%% @doc The following operation verifies
+%%      that Operation is supported by this particular CRDT.
+-spec is_operation(term()) -> boolean().
+is_operation(Operation) ->
```

Comment:
done in riak_dt.

> On Sep 23, 2015, at 15:06, Alejandro Z. Tomsic <aletomsic@gmail.com> wrote:
> 
> Done in antidote, will continue in riak_dt.
> Also fixed the failing test.
> 
> 
>> On Sep 23, 2015, at 13:17, Annette Bieniusa <notifications@github.com <mailto:notifications@github.com>> wrote:
>> 
>> In src/crdt_bcounter.erl <https://github.com/SyncFree/antidote/pull/142#discussion_r40192109>:
>> 
>> > @@ -154,6 +155,28 @@ to_binary(C) -> term_to_binary(C).
>> >  -spec from_binary(binary()) -> bcounter().
>> >  from_binary(<<B/binary>>) -> binary_to_term(B).
>> >  
>> > +%% @doc The following operation verifies
>> > +%%      that Operation is supported by this particular CRDT.
>> > +-spec is_operation(term()) -> boolean().
>> > +is_operation(Operation) ->
>> Please use andalso/2 instead of and/2 for short-circuiting here (and in the other type checks)!
>> 
>> —
>> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/142/files#r40192109>.
>> 
>